axelrod/strategies/probabilistic_hill_climb.py

```diff
@@ -0,0 +1,55 @@
+from axelrod.action import Action
+from axelrod.player import Player
+
+C, D = Action.C, Action.D
+
+class ProbabilisticHillClimb(Player):
+	"""
+	Defects with probability of 50%.
+	Every time the oppenent deffects, probability becomes '(100 + 1)/100', increasing by 1%.
+	Every time the opponent confesses, probability becomes '(100 - 1)/100', decreasing by 1%.
```

Comment:
The `C` in the IPD stands for cooperate (confessing is technically the defection).
```suggestion
	Every time the opponent cooperates, probability becomes '(100 - 1)/100', decreasing by 1%.
```

Reply:
(This will need to be modified throughout.)